corehq/ex-submodules/couchforms/getters.py

```diff
@@ -54,6 +69,21 @@ def get_instance_and_attachment(request):
     return instance, attachments
 
 
+def _valid_xml_extension(file):
+    return _valid_file_extension(file, PERMITTED_FORM_SUBMISSION_FILE_EXTENSIONS)
+
+
+def _valid_attachment_extension(file):
+    return _valid_file_extension(file, SUPPORTED_MEDIA_FILE_EXTENSIONS)
+
+
+def _valid_file_extension(file, permitted_extensions):
+    if "." not in file.name:
+        return False
+    file_extension = file.name.rsplit('.', 1)[-1]
```

Comment:
nit: (consistency with rest of function)
```suggestion
    file_extension = file.name.rsplit(".", 1)[-1]
```

Reply:
thanks @snopoke , missed committing that